packages/atomic/src/components/result-template-components/atomic-result-link/atomic-result-link.tsx

```diff
@@ -41,6 +41,7 @@ export class AtomicResultValue {
         part="result-link"
         href={filterProtocol(this.result.clickUri)}
         target={this.target}
+        class="text-lg"
```

Comment:
This won't work because we don't add tailwind in its entirety in prod, might work only in dev. That should go inside the default.html result template

Reply:
Maybe we could enable the Shadow DOM and inject Tailwind in this component and the printable URI one, I forgot about the limitation too in the PR  https://github.com/coveo/ui-kit/pull/708

Reply:
So... I'm not sure I understand the limitation here ?

We can't use tailwind in *any* result component ? 🤔  Or we can only use "some" tailwind attributes ? 
Otherwise, all the styling has to be done externally in each and every single result template ?

That seems like... a huge problem ?

Reply:
Or, could we just add a .pcss file for `atomic-result-link` where we inject that syling ?